### PR TITLE
Define a default progress bar format

### PR DESCRIPTION
**Pull request by gautierronan**
With this PR, the text after the tqdm progress bar looks like
```python
37.3% - Time 00:11/00:19
```

### CODE REVIEW

**Comment by abocquet** [comment]
Nice initiative. To this point maybe we could directly code a `torchqdiffeq.tqdm` function that calls tqdm with the correct formatting to avoid copy pasting it everywhere and be easier to add more options later ?

**Comment by gautierronan** [comment]
> Nice initiative. To this point maybe we could directly code a `torchqdynamics.tqdm` function that calls tqdm with the correct formatting to avoid copy pasting it everywhere and be easier to add more options later ?

Yes maybe something more like the last commit?

**Review by abocquet — DISMISSED** [review]
Yes, this is exactly what I had in mind

File: torchqdynamics/progressbar.py
Comment (PierreGuilmin) [inline torchqdynamics/progressbar.py:1]: Could we rename this file `progress_bar.py` to follow our naming convention?

File: torchqdynamics/progressbar.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from functools import partial
+
+from tqdm.auto import tqdm as std_tqdm
```
Comment (PierreGuilmin) [inline torchqdynamics/progressbar.py:3]: I'm really not fan of the notebook progress bar viewer (too big and colorfull IMO, plus it doesn't work well with dark mode), could we switch to have the regular progress bar everywhere (`from tqdm import tqdm`)?

Screenshot 2023-04-03 at 16 31 40

File: torchqdynamics/progressbar.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+# Define a default progress bar format
+PBAR_FORMAT = '{desc}: {percentage:3.1f}% - Time {elapsed}/{remaining}'
+
+# Redefine tqdm with some default arguments
```
Comment (PierreGuilmin) [inline torchqdynamics/progressbar.py:8]:
```suggestion
# redefine tqdm with some default arguments
```

**Review by PierreGuilmin — CHANGES_REQUESTED** [review]
Nice implem!

**Comment by gautierronan** [comment]
In the end I went with `|██████████| 100.0% - time 00:05/00:00`, and removed the `{desc}` since it's not implemented yet.

**Pull request merged**